src/Proto.Cluster/Partition/PartitionIdentityActor.cs

```diff
@@ -687,7 +687,7 @@ TaskCompletionSource<ActivationResponse> setResponse
                             }
 
                             var currentActivatorAddress =
-                                _cluster.MemberList.GetActivator(msg.Kind, context.Sender!.Address)?.Address;
+                                _cluster.MemberList.GetActivator(msg.Kind, context.Sender!.Address, msg.ClusterIdentity)?.Address;
```

Comment:
My only nitpick is that we now get "kind" twice.
once as a string, and once in the cluster identity.

We always have the full identity in all cases, right?
I´m thinking if we could just. tweak the GetActivator with an overload that just take address and identity (?)

and Obsolete warn on the other one.
I can´t imagine there are many custom member strategies out there in the wild anyway.